autoload/ale/fixers/rubocop.vim

```diff
@@ -9,7 +9,7 @@ function! ale#fixers#rubocop#GetCommand(buffer) abort
     return ale#ruby#EscapeExecutable(l:executable, 'rubocop')
     \   . (!empty(l:config) ? ' --config ' . ale#Escape(l:config) : '')
     \   . (!empty(l:options) ? ' ' . l:options : '')
-    \   . ' --auto-correct --force-exclusion %t'
+    \   . ' --auto-correct-all --force-exclusion %t'
 endfunction
```

Comment:
Let's make this optional, and use `--auto-correct` as the default. Reading the documentation, they say that some fixes are safe, and others are unsafe, so let's use only the safe fixes by default.

Reply:
Is there a way to fix this in my config in the meantime? Setting the option in my vim config thus:

```vim
let g:ale_ruby_rubocop_options = '--auto-correct-all'
```

results in a rubocop command like:

```
 ['/usr/bin/zsh', '-c', '''rubocop'' --config ''.../.rubocop.yml'' --auto-correct-all --auto-correct --force-exclusion ''/tmp/nvim7B7OMS/.../myfile.rb''']
```

It seems the `--auto-correct-all` I provided comes first, so the `--auto-correct` default here "wins". Could we swap the order that `options` occurs to after the defaults in the command line?

```vim
    return ale#ruby#EscapeExecutable(l:executable, 'rubocop')
    \   . (!empty(l:config) ? ' --config ' . ale#Escape(l:config) : '')
    \   . '--auto-correct --force-exclusion'
    \   . (!empty(l:options) ? ' ' . l:options : '')
    \   . '%t'
```

Reply:
@w0rp I have made the requested changes. Now safe auto correction is by default. Users can enable non safe autocorrection by setting `ruby_rubocop_auto_correct_all` variable:

```
let g:ale_ruby_rubocop_auto_correct_all = 1
```